### DISCUSSION TITLE
How to supply commands and arguments

### DISCUSSION THREAD

**Discussion by ghost** [rasta-mouse/OST-C2-Spec#4] [Ideas]
Would it be reasonable to specify an Ideal on how commands and arguments should be supplied on a command line interface or should that be decided by the implementer.

for example:
```
/<type> /<code> <arguments>
```

**Comment by rasta-mouse** [comment]
The spec is only concerned about the message structure, not these kinds of implementation details.  There is also no reason to expose the type or code to the user at all.
